<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatCreationWindow</class>
 <widget class="QDialog" name="ChatCreationWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>368</width>
    <height>439</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Новый чат</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>321</width>
     <height>391</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="personal_radioButton">
        <property name="text">
         <string>Личный чат</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="group_radioButton">
        <property name="text">
         <string>Групповой чат</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLineEdit" name="nazv_lineEdit">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListView" name="users_listView">
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="create_pushButton">
      <property name="text">
       <string>Создать</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
